typings/wordpress__data.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@wordpress/data' {
+	export * from '@wordpress/data/build-types/index';
```

Comment:
how does this play with the issue of having a global DefinitelyTyped package in for `@wordpress/data`?

Reply:
This overrides the DefinitelyTyped styles thanks to the `"typeRoots": [ "./typings", "./node_modules/@types" ],` setting in tsconfig.json.